source/sdk/math.ooc

```diff
@@ -358,6 +318,9 @@ extend Float {
 }
 
 extend LDouble {
+	pi ::= static 3.14159_26535_89793_23846_26433_83279 as This
+	e ::= static 2.718281828459045235360287471352662497757247093699959574966 as This
```

Comment:
Okay, but does this actually have LDouble precision, or is it truncated to a Double before being cast to an LDouble? (Not that it matters, but I'm curious.)